handsontable/src/plugins/copyPaste/copyPaste.js

```diff
@@ -91,35 +88,39 @@ export class CopyPaste extends BasePlugin {
   }
 
   /**
-   * Maximum number of columns than can be copied to clipboard using <kbd>**Ctrl**</kbd>/<kbd>**Cmd**</kbd> + <kbd>**C**</kbd>.
+   * The maximum number of columns than can be copied to the clipboard.
    *
    * @type {number}
    * @default Infinity
    */
   columnsLimit = Infinity;
+
```

Comment:
```suggestion
```
I guess the Team applies the rule of no new lines between class fields. Can you revert that?